backend/src/main/java/com/redhat/cloud/notifications/models/EndpointProperties.java

```diff
@@ -25,7 +25,7 @@
     private UUID id;
 
     @MapsId
-    @OneToOne(fetch = LAZY)
+    @OneToOne(fetch = EAGER) // Todo: This was causing issues on the stateless session createEndpoint - Revisit
```

Comment:
This was failing here: https://github.com/RedHatInsights/notifications-backend/blob/bf31e44617c4e84973c8ee3794beae218483a00a/backend/src/main/java/com/redhat/cloud/notifications/db/EndpointResources.java#L51